problems/1.types+operators.js

```diff
@@ -2,15 +2,20 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+const isNumber = (val) => {
+  let(val < 23)
+return "is val a number"
```

Comment:
This should return a boolean. You're returning a number. Think about using typeof to help you determine, to see if its' greater than 1. Numbers less than 23 are still numbers. let is for variable declaration.